.travis.yml

```diff
@@ -143,6 +143,8 @@ install:
 - conda info -a
 - conda create -q -n p4env python=$PYTHON_VER numpy cmake ci-psi4-lt psi4-lt-mp gau2grid dftd3 gcp snsmp2 scipy -c psi4/label/dev -c psi4
 - source activate p4env
+- conda install hungarian networkx -c psi4
+- conda install deepdiff -c conda-forge
```

Comment:
Are we adding this as Psi4 requirements?

Reply:
No. hungarian and networkx are very specialized, and they're only needed if you want to align two molecules and can't guarantee that the atoms map (first atom of molA plays same structural role as first atom of molB, etc.) So those are definitely in-function imports so that run-of-the-mill calcs aren't burdened.

deepdiff you actually will hit by default at present b/c there's a temporary? internal check that the `Mol.to_dict` are sound. It's easy enough to get, as I added it to conda-forge, but what do you think? Require or disable internal checks and some test cases if not present?

In fact, right now you have to compile hungarian yourself for most platforms. Author would like to shove it into numpy as his is in C and reputedly far faster than the np implementation. But no one's actively working on that. Btw, @dgasmith, do you know how a [module](https://github.com/hrldcpr/hungarian/blob/master/hungarian.cpp#L8) that compiles against `numpy/arrayobject.h` chooses its numpy? My py27 conda pkg has been supplied with a np112 numpy but ends up with np112 in its conda build string but np110 in its setup.py build string. And complains at runtime about being compiled against a diff version. There's `NPY_1_7_API_VERSION` warnings at compile-time if that's a clue.

Reply:
Anything we can `pip` install I wouldn't worry too much about; however, if we start requiring libraries from conda-forge we *enforce* conda on everyone. That is a step I am not too comfortable with at the moment. As a note, you can get networkx/deepdiff off pip.

If he wants to get a C hungarian into NumPy, I can help him with that.

Yea, no idea on `setup/distutils` compilation, I try to stay far away.